conf/openimis_win.conf

```diff
@@ -0,0 +1,110 @@
+upstream docker-ibackend {
+        server ibackend:8000;
+}
+upstream docker-backend {
+        server backend:8000;
+}
+
+lua_shared_dict sessions 10m;     
+server {
+        listen 80 default_server;
+        server_name _;
+        return 301 https://$host$request_uri;
+}
+server {
+        listen       443 ssl;
+        server_name NEW_OPENIMIS_HOST;
+
+        ssl_certificate C:\certs\localhost.crt;
+        ssl_certificate_key C:\certs\localhost.key;
+
+
+        location /.well-known {
+                root C:\openresty\html;
+        }
+
+        location / {
+                proxy_pass http://LEGACY_OPENIMIS_HOST;
+                proxy_set_header   Host LEGACY_OPENIMIS_HOST;
+                proxy_set_header   X-Real-IP $remote_addr;
+                proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header   X-Forwarded-Host $server_name;
+                lua_need_request_body on;
+                log_by_lua_file C:\openresty\lua\login_interceptor.lua;                
+        }
+
+        location /photos/ {
+                proxy_pass http://LEGACY_OPENIMIS_HOST/;
+                proxy_set_header   Host LEGACY_OPENIMIS_HOST;
+                proxy_set_header   X-Real-IP $remote_addr;
+                proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_set_header   X-Forwarded-Host $server_name;
+        }
+
+        location /Manual/IMIS_manual.pdf {
```

Comment:
Do you have to put also the filename? I added the FR version of the manual also and with this, it will not be accessible.

Reply:
no but I think this line is useless because by default it will go to legacy, TBC